packages/devtools/lib/src/tables.dart

```diff
@@ -556,15 +695,93 @@ abstract class Column<T> {
   String toString() => title;
 
   void renderToElement(CoreElement cell, T dataObject) {
-    final String content = render(getValue(dataObject));
+    final String content = render(getDisplayValue(dataObject));
     if (usesHtml) {
       cell.setInnerHtml(content);
     } else {
       cell.text = content;
     }
+    cell.tooltip = getTooltip(dataObject);
+  }
+
+  String _getAlignmentCss(ColumnAlignment alignment) {
+    switch (alignment) {
+      case ColumnAlignment.left:
+        return 'left';
+      case ColumnAlignment.right:
+        return 'right';
+      case ColumnAlignment.center:
+      default:
+        return 'center';
+    }
   }
 }
 
+abstract class TreeColumn<T extends TreeTableNode> extends Column<T> {
+  TreeColumn(
+    title, {
+    int fixedWidthPx,
+    double percentWidth,
```

Comment:
nit: fractionWidth not percent width.
Looks like the values are .8 and 1.0 so they are already fractions. You could also adopt the naming conventions and semantics from flex layouts if it made sense.